### ISSUE TITLE
iframe don't update

### ISSUE THREAD

**Issue by Escartem** [electron/electron#24530]
### Preflight Checklist
I'm having an issue with iframes in electron, the iframe don't seem to update until the app is reinstalled

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  *  v9.0.5
* **Operating System:**
  *  Windows 10 (1903)

### Expected Behavior

The iframe should be reloaded on every launch or reload

### Actual Behavior

The iframe doesn't reload or show the new content when reloading.
Installing the app again seems to fix the bug

### To Reproduce

create an app containing an iframe for example website.com
compile the app, install it and launch it
update the iframe website ( like adding text inside )
reload or relaunch the app

[Repository](https://github.com/Escartem/Frame) of my project where I'm having this issue

**Comment by Escartem** [comment]
update : 
The iframe is updating but only after ~30 minutes after the website has changed

**Comment by jkleinsc** [comment]
Please provide a standalone test that a tester could run to reproduce the issue. This makes fixing issues go more smoothly: not only does it ensure the right thing is tested, but it also prevents triage from bottlenecking on making testers write new tests for each new issue.

I'm setting the more-information-needed label for the above reasons. After you've responded, please @ me in a followup comment.

Thanks in advance! Your help is appreciated.

**Comment by Escartem** [comment]
Okay, here is an example project ready to be built that can be downloaded [here](https://github.com/electron/electron/files/4926720/electron-quick-start.zip).

A website that can be edited is required ( like a github page website ) and just add some text inside like _Hello World_

How to reproduce the issue:

1. Download the project
1.1 Install node modules
2. Edit the iframe in `index.html` to show the website that will be edited
3. Run `npm run dist` to build an installer
4. Run the installer
5. Launch the app to load the iframe
6. Edit the website text ( like _Hello World 2_ )
6.1 Wait for the website to update
7. Launch the app again and the iframe should still show _Hello World_ and will only show _Hello World 2_ after ~20 minutes

Hope this help !
@jkleinsc

**Comment by jkleinsc** [comment]
@Escartem that sounds like a caching issue.  Have you tried clearing the http cache, eg: https://www.electronjs.org/docs/api/session#sesclearcache

**Comment by jkleinsc** [comment]
@Escartem I tested this locally and wasn't able to reproduce.  I do have one thought though -- if you want to make sure the iframe has the latest contents in it, you should reload the iframe which you can do via setting the iframe src, eg:
```
        const myframe = document.getElementById('myframe');
        myframe.src = "http://localhost:8000/hello.html"
```

**Comment by jkleinsc** [comment]
@Escartem given the nature of your issue here it doesn't appear to be a bug, I would recommend checking out the [Electron community](https://github.com/electron/electron#community). There are also a bunch of helpful people in this [community forum](https://discuss.atom.io/c/electron) that should be willing to point you in the right direction.
